wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -1767,6 +1770,31 @@ instance BuildableSafeGen NodeInfo where
         nfoLocalTimeInformation
         (Map.toList nfoSubscriptionStatus)
 
+-- | A shielded redemption code.
+newtype ShieldedRedemptionCode = ShieldedRedemptionCode
+    { unShieldedRedemptionCode :: Text
+    } deriving (Eq, Show, Generic)
```

Comment:
What meaning do we give to `Shielded` here? Is that because of the newtype wrapping the basic `Text` type?
If so, I find it a bit confusing ^^" ... What about simply `RedemptionCode` ?

Reply:
The Daedalus codebase refers to it as the shielded redemption code frequently. Or, a shielded vending key:

```
λ ~/Projects/daedalus/ develop ag shieldedRedemptionCode
source/renderer/app/i18n/locales/hr-HR.json
269:  "wallet.redeem.dialog.shieldedRedemptionCodeError": "!!!Invalid shielded vending key",

source/renderer/app/i18n/locales/zh-CN.json
269:  "wallet.redeem.dialog.shieldedRedemptionCodeError": "!!!Invalid shielded vending key",

source/renderer/app/i18n/locales/de-DE.json
269:  "wallet.redeem.dialog.shieldedRedemptionCodeError": "!!!Invalid shielded vending key",

source/renderer/app/i18n/locales/defaultMessages.json
1684:        "id": "wallet.redeem.dialog.shieldedRedemptionCodeError",

source/renderer/app/i18n/locales/ko-KR.json
269:  "wallet.redeem.dialog.shieldedRedemptionCodeError": "!!!Invalid shielded vending key",

source/renderer/app/i18n/locales/ja-JP.json
269:  "wallet.redeem.dialog.shieldedRedemptionCodeError": "無効な保護還元キー",

source/renderer/app/i18n/locales/en-US.json
269:  "wallet.redeem.dialog.shieldedRedemptionCodeError": "Invalid shielded vending key",

source/renderer/app/components/wallet/ada-redemption/AdaRedemptionForm.js
126:    id: 'wallet.redeem.dialog.shieldedRedemptionCodeError',
```

Given that the original datatype in the original wallet just said `seed :: Text -- TODO: newtype this`, I made a best guess and am happy to choose something more meaningful :)